docs/requirements.md

```diff
@@ -1,28 +1,29 @@
-# Requirements in the Underlying Databases of ScalarDB
+# Requirements and Recommendations for the Underlying Databases of ScalarDB
 
-This document explains the requirements in the underlying databases of ScalarDB to make ScalarDB applications work correctly.
+This document explains the requirements and recommendations in the underlying databases of ScalarDB to make ScalarDB applications work correctly.
 
-## Cassandra (or Cassandra compatible databases)
+## Cassandra or Cassandra-compatible database requirements
```

Comment:
It's just a heads-up.
We also need a requirements section for CosmosDB since we need to set `strong` for consistency?

Reply:
Yeah, that's correct. Thank you for the heads-up!

Reply:
I've added a section about this to lines [21-25](https://github.com/scalar-labs/scalardb/pull/1020/files#diff-71c6b87612afd48ee2b295f885d377d87f291385b3b5ff796e71947739393994R21-R25). PTAL!